.config/typos.toml

```
@@ -5,6 +5,8 @@ extend-exclude = [
     "deps/",
     # crc16_slottable is primarily pre-generated random strings.
     "src/crc16_slottable.h",
+    # 00-RELEASENOTES includes non-English names that might be flagged as typos in other contexts.
+    "00-RELEASENOTES"
```

this way, when we add new lines later, we don't need to touch this line.
```suggestion
    "00-RELEASENOTES",
```